docs/pages/components/[slug]/code.tsx

```diff
@@ -34,9 +34,13 @@ export default function PackagesCode({
 							href: '#pkg-content',
 						},
 					]}
-					editPath={`/components/${pkg.slug}/docs/code.mdx`}
+					editPath={`packages/react/src/${pkg.slug}/docs/code.mdx`}
 				>
 					<Prose id="pkg-content">
+						<h2>Usage</h2>
+						<pre>
+							<code>{`import { ... } from '@ag.ds-next/react/${pkg.slug}';`}</code>
```

Comment:
Should it be:
```suggestion
							<code>{`import { ${pkg.title} } from '@ag.ds-next/react/${pkg.slug}';`}</code>
```
Or are there edge-cases where this doesn't work?